server/src/main/java/com/linecorp/centraldogma/server/CentralDogma.java

```diff
@@ -751,10 +751,17 @@ public String serviceName(ServiceRequestContext ctx) {
                                    .build(),
                         "/index.html"));
             }
-            sb.serviceUnder("/",
+
+            sb.serviceUnder("/legacy-web",
                             FileService.builder(CentralDogma.class.getClassLoader(), "webapp")
                                        .cacheControl(ServerCacheControl.REVALIDATED)
                                        .build());
+
+            sb.serviceUnder("/",
+                            FileService.builder(CentralDogma.class.getClassLoader(),
+                                                "com/linecorp/centraldogma/webapp")
```

Comment:
If a Central Dogma server starts with Gradle, the HTML and JS files are automatically added to classpath and served.